### ISSUE TITLE
Permission denied when trying to write to nornir.log

### ISSUE THREAD

**Issue by walidamer711** [nornir-automation/nornir#257]
I am getting a permission error as the following
Unable to configure handler 'info_file_handler': [Errno 13] Permission denied: '/nornir.log'
when the nornir is trying to call /usr/lib/python3.6/logging/config.py 

                            handler.name = name
                            handlers[name] = handler
                        except Exception as e:
                            if 'target not configured yet' in str(e):
                                deferred.append(name)
                            else:
                                raise ValueError('Unable to configure handler '
                                                 '%r: %s' % (name, e))

any idea how resolve this.

**Comment by dbarrosop** [comment]
check the configuration section of the documentation to change the location of the log file or disable it altogether.

I am closing this issue as this is problem is specific to your environment but feel free to comment if you have further doubts.

**Comment by ktbyers** [comment]
It looks like you are trying to write to the root file-system. Have you tried using './nornir.log'?



Kirk

**Comment by walidamer711** [comment]
Note: I am running nornir from django view.py function.
I have tried that (logging_file: "./nornir.log") but it didn't work and still the same permission deny error, however, when I added the full path it worked fine. (logging_file: "/home/wamer/nornir.log")
I have faced the same issue with jinja templates location as well. so only when specify the full path django could be able to locate the template.
